test/ClangImporter/availability.swift

```diff
@@ -1,4 +1,4 @@
-// RUN: %target-swift-frontend(mock-sdk: %clang-importer-sdk) -typecheck -verify -I %S/Inputs/custom-modules %s
+// RUN: %target-swift-frontend(mock-sdk: %clang-importer-sdk) -typecheck -verify -I %S/Inputs/custom-modules %s -verify-ignore-unknown
```

Comment:
Where do the unknown locations come from? Is that for notes on the imported accessors?

Reply:
Yep. I'm not super happy about that but it seems unrelated to the purpose of this PR.